### PR TITLE
Update PR template to prompt for accessibility

### PR DESCRIPTION
**Pull request by marcysutton**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [X] Optimization
- [X] Documentation Update

## Related Tickets & Documents

Closes https://github.com/forem/forem/issues/11406. I'm also open to suggestions on what to add to the template, if it's too heavy-handed! 

## QA Instructions, Screenshots, Recordings

Mostly docs changes, so they can be reviewed in this PR and locally.

## Added tests?

- [ ] Yes
- [X] No, and this is why: docs-only changes
- [ ] I need help with writing tests

## Added to documentation?

- [X] [Developer Docs](https://docs.forem.com) and/or [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [X] README
- [ ] No documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Doing the work! an animated GIF

### CODE REVIEW

File: docs/frontend/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Accessibility
+---
+
+# Accessibility
+
+To make Forem the most inclusive community platform around,accessibility should be considered to enable people with disabilities to create and consume content.
```
Comment (rhymes) [inline docs/frontend/accessibility.md:7]: missing space between `around,` and `accessibility`

**Review by rhymes — COMMENTED** [review]
I love it! I wonder if we can integrate https://docs.forem.com/tests/accessibility-tests/ somewhere in the new document which contains further info

**Review by jacobherrington — APPROVED** [review]
Great addition @marcysutton :+1: 

Other than the quick spacing typo rhymes mentioned, I think this is good to go! :smile:

**Comment by marcysutton** [comment]
Thanks for the review, I've incorporated your feedback and fixed a merge conflict in the README. Hopefully it's good to go!

**Review by vaidehijoshi — APPROVED** [review]
_just lovely_ 😃

**Review by rhymes — APPROVED** [review]
LGTM!

**Pull request merged**